### PR TITLE
#169520091 Add view entries endpoint

### PR DESCRIPTION
**Pull request by T2Wil**
#### What does this PR do?
- Add view entries endpoint
#### Description of Task to be completed?
- Add view entries controller
- Add database queries
- Implement tests
#### How should this be manually tested?
* have Postman
* clone the repository
* cd into directory
* run ``npm run start:dev``
* POST `api/v2/viewentries`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#169520091
#### Screenshots (if appropriate)
`INPUT VALID TOKEN`

image

`ENTRIES IN DB`

image

### CODE REVIEW

File: server/controllers/entriesController.js [hunk 1]
```diff
@@ -1,115 +1,70 @@
...
-        userId,
-        specificEntry,
-      },
+      totalEntries,
+      pages: `Page ${page} of ${totalPages}`,
+      userId,
+      data: entries,
     });
-  } else {
-    res.status(404).json({
```
Comment (mcaleb808) [inline server/controllers/entriesController.js:91]: @T2Wil what happens when a user doesn't have any entry?

**Review by mcaleb808 — CHANGES_REQUESTED** [review]

File: server/controllers/entriesController.js (hunk 1 above)
Comment (T2Wil) [reply server/controllers/entriesController.js:91]: @mcaleb808
`Created a new user`
image

`Signed in `
image


`Use the token to view entries`
image

Comment (mcaleb808) [reply server/controllers/entriesController.js:91]: @T2Wil Please also add scenario where a user is notified that he doesn't have any entry

Comment (T2Wil) [reply server/controllers/entriesController.js:91]: @mcaleb808
Added Not found status, and an error message
image

**Review by mcaleb808 — APPROVED** [review]

**Pull request merged**